From a3c6e92dfb752deb7f60342f0fd379bfed895eeb Mon Sep 17 00:00:00 2001
From: Craig Tatlor <ctatlor97@gmail.com>
Date: Mon, 21 May 2018 18:21:42 +0100
Subject: [PATCH 2/3] wpa_supplicant_8_lib: Added LOCAL_VENDOR_MODULE to set
 output path of the binaries

Makes binaries build to /vendor.

Change-Id: Id749e80d4949404feb70116ad509ece6cbff15b2
---
 bcmdhd/wpa_supplicant_8_lib/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bcmdhd/wpa_supplicant_8_lib/Android.mk b/bcmdhd/wpa_supplicant_8_lib/Android.mk
index 5dc4c9f..6a08139 100644
--- a/bcmdhd/wpa_supplicant_8_lib/Android.mk
+++ b/bcmdhd/wpa_supplicant_8_lib/Android.mk
@@ -64,6 +64,7 @@ endif
 
 include $(CLEAR_VARS)
 LOCAL_MODULE := lib_driver_cmd_bcmdhd
+LOCAL_VENDOR_MODULE := true
 LOCAL_SHARED_LIBRARIES := libc libcutils
 LOCAL_CFLAGS := $(L_CFLAGS)
 LOCAL_SRC_FILES := $(WPA_SRC_FILE)
-- 
2.17.1

